pkg/gofr/datasource/pubsub/mqtt/mqtt.go

```diff
@@ -322,6 +322,11 @@ func (m *MQTT) Unsubscribe(topic string) error {
 	return nil
 }
 
+func (m *MQTT) Close() error {
+	m.Client.Disconnect(250)
```

Comment:
250 is a magic number here

Please add a variable or a const and a comment about its meaning